skipper-go/contracts/src/Multihop.sol

```diff
@@ -35,7 +39,7 @@ contract Multihop is Ownable {
     function swapMultihop(
         address fromToken,
         uint256 fromAmount,
-        DexHop[] memory route
+        DexHop[] calldata route
```

Comment:
route is not mutated in the function body so using calldata would save some gas.